packages/rum/test/benchmarks/profiler.js

```diff
@@ -91,9 +103,17 @@ function gatherRawMetrics(browser, url) {
       Object.assign(metrics, timings)
     })
     /**
-     * Start the profiler before navigating to the URL
+     * Perform a garbage collection to do a clean check everytime
+     */
+    await client.send('HeapProfiler.collectGarbage')
```

Comment:
Why is this necessary? Don't we start with a new window every time?

Reply:
Just to make sure memory is not reused, yes we do start a new window, but just in case.